src/components/ThemeButton.vue

```diff
@@ -15,9 +14,9 @@ watchEffect(() => {
 </script>
 
 <template>
-	<button class="btn btn-circle btn-ghost swap" @click="toggleDark()">
```

Comment:
As this component is not used anywhere else, I think we can integrate it inside the `UserDropdown` component you created :)